doc/source/operator_reference.rst

```diff
@@ -32,9 +23,17 @@ Click below to access the operators documentation.
                :click-parent:              
 
 
+For Ansys 2023 R2 and later, the DPF Server licensing logic for operators in DPF depends on the active
+server context.
```

Comment:
why remove the link to the `ServerContext` object?

Reply:
@PProfizi It is broken:
https://dpf.docs.pyansys.com/api/ansys.dpf.core.server_context.html#servercontext

I just get a 404 error. Can you add the correct link here and I will re-add it?

Reply:
@PProfizi Yes, thanks!